internal/target/container_target.go

```diff
@@ -0,0 +1,17 @@
+package target
+
+type ContainerTargetOptions struct {
+	Filename  string `json:"filename"`
+	Reference string `json:"reference"`
+
+	Username string `json:"username"`
+	Password string `json:"password"`
+
+	TlsVerify *bool `json:"tls_verify,omitempty"`
+}
+
+func (ContainerTargetOptions) isTargetOptions() {}
+
+func NewContainerTarget(options *AzureTargetOptions) *Target {
```

Comment:
Is this `AzureTargetOptions` a copy paste mistake?

Reply:
Most definitely is ;)